components/wifi/COMPONENT_QCA400X_WIFI/QCA_WiFiInterface.h

```diff
@@ -0,0 +1,378 @@
+/* QCA4002/4004 implementation of NetworkInterfaceAPI
+ * Copyright (c) 2019 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef QCA_WIFIINTERFACE_H
+#define QCA_WIFIINTERFACE_H
+
+#include "mbed.h"
```

Comment:
`mbed.h` should be used only by applications and not inside Mbed OS itself.

Reply:
Updated and removed this include